app/webpacker/src/stylesheets/_commodity-tree.scss

```diff
@@ -570,3 +570,7 @@ article.tariff {
 .non-breaking-heading {
   white-space: nowrap;
 }
+
+.quotas-blurb {
```

Comment:
We have so much special snowflake css styles :(

Reply:
Highly specific classes like this should be a real edgecase we try to avoid at all costs

If we need custom css, we should have it as part of a generic element, eg something like `small-text`

_but_ that actuall already exists and has a font-size of 16px, its `govuk-body-s`

see https://design-system.service.gov.uk/styles/typography/#body-small